### PR TITLE
Mobile Release v1.52.1

### PR DESCRIPTION
**Pull request by fluiddot**
## Description
Release 1.52.1 of the react-native-editor and Gutenberg-Mobile.

For more information about this release and testing instructions, please see the related Gutenberg-Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/3467

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [ ] I've tested my changes with keyboard and screen readers. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR (please manually search all `*.native.js` files for terms that need renaming or removal).

### CODE REVIEW

**Comment by fluiddot** [comment]
Looks like the PR has conflicts with `trunk` 😞  but since we haven't modified any file on Gutenberg, we should only see the version changed in the different packages related to Gutenberg-mobile.

**Comment by dcalhoun** [comment]
> Looks like the PR has conflicts with `trunk` 😞 but since we haven't modified any file on Gutenberg, we should only see the version changed in the different packages related to Gutenberg-mobile.

The same type of conflicts occurred for https://github.com/WordPress/gutenberg/pull/31242#pullrequestreview-645937130 as well. I believe the conflicts originate due to the nature of a beta fix branching off a tag, which is a merge commit. The changes in the PR LGTM so far.

**Review by dcalhoun — APPROVED** [review]
LGTM. Smoke tested on a iPhone SE (build `47372`) and Samsung Galaxy S20 (build `105077`).

File: packages/react-native-editor/CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,12 @@ For each user feature we should also add a importance categorization label  to i
 
 ## 1.52.0
 
+## 1.52.1
+
+- [*] Fixes for the generated localized strings files.
+
+## 1.52.0
+
```
Comment (fluiddot) [inline packages/react-native-editor/CHANGELOG.md:23]: Looks like we duplicated the entry for `1.52.0`.

Comment (dcalhoun) [reply packages/react-native-editor/CHANGELOG.md:23]: Bummer. Sorry about that. I will open a PR to clean it up.

Comment (fluiddot) [reply packages/react-native-editor/CHANGELOG.md:23]: No worries, let me know if you create the PR. I could also update the file if you’re working on other tasks.

Thank you very much @dcalhoun for wrangling the release ❤️, you did an amazing job!

**Pull request merged**